py/client-ticking/build.gradle

```diff
@@ -56,47 +58,84 @@ def buildPyClientTicking = Docker.registerDockerTask(project, 'pyClientTicking')
         copyFile('README.md', "${prefix}/src/py-client-ticking")
         copyFile('src/', "${prefix}/src/py-client-ticking/src/")
         copyFile('wheels/', "${prefix}/src/py-client-ticking/in-wheels")
-        runCommand("PREFIX=${prefix}; DEEPHAVEN_VERSION=${project.version};" +
+        runCommand("PREFIX=${prefix}; WHEELS_SET='${wheelsSet}'; DEEPHAVEN_VERSION=${project.version};" +
                    '''set -eux ; \
                      cd "${PREFIX}/src/py-client-ticking"; \
                      . "${PREFIX}/env.sh"; \
-                     MAKEFLAGS="-j${NCPUS}" \
-                       CFLAGS="-I${DHCPP}/include" \
-                       LDFLAGS="-L${DHCPP}/lib" \
-                       DEEPHAVEN_VERSION="${DEEPHAVEN_VERSION}" \
-                       python3 setup.py build_ext -i; \
-                     DEEPHAVEN_VERSION="${DEEPHAVEN_VERSION}" python3 setup.py bdist_wheel; \
-                     pip3 install in-wheels/*.whl; \
-                     pip3 install --force --no-deps dist/*.whl; \
+                     ORIG_PATH=$PATH; \
+                     for py in ${WHEELS_SET}; do \
+                       rm -f *.cpp *.so; \
+                       PATH="/opt/python/${py}/bin:$ORIG_PATH"; \
+                       pip3 install cython; \
+                       MAKEFLAGS="-j${NCPUS}" \
+                         CFLAGS="-I${DHCPP}/include" \
+                         LDFLAGS="-L${DHCPP}/lib" \
+                         DEEPHAVEN_VERSION="${DEEPHAVEN_VERSION}" \
+                         python3 setup.py build_ext -i; \
+                       DEEPHAVEN_VERSION="${DEEPHAVEN_VERSION}" python3 setup.py bdist_wheel; \
+                       auditwheel repair dist/pydeephaven_ticking*${py}*.whl; \
+                       rm -f dist/pydeephaven_ticking*${py}*.whl; \
+                       mv -f wheelhouse/*.whl dist; \
+                       pip3 install in-wheels/*.whl; \
+                       pip3 install --force --no-deps dist/pydeephaven_ticking*${py}*.whl; \
+                     done; \
                      ln dist/*.whl /out; \
                      cd /; \
                      rm -fr "${PREFIX}/src/py-client-ticking"
                    ''')
 
         environmentVariable 'LD_LIBRARY_PATH', ""  // avoid conflict with libarrow.13.0.0.so
     }
-    parentContainers = [ project.tasks.getByPath(':cpp-client:cppClient') ]
-}
+    parentContainers = [ project.tasks.getByPath(':cpp-client:cppClientPy') ]
+}}
+
+def buildPyClientTicking = buildPyClientTickingManyLinux("cp38-cp38", 'pyClientTicking')
+
+def wheelsSet = [ 'cp38-cp38', 'cp39-cp39', 'cp310-cp310', 'cp311-cp311', 'cp312-cp312' ]
+def buildPyClientTickingAllWheels = buildPyClientTickingManyLinux(String.join(' ', wheelsSet), 'pyClientTickingAllWheels')
```

Comment:
Just to make sure I'm seeing this correctly; we are testing 3.8, but not 3.9+?

Reply:
Yeah, my thought here was that we don't want to tax everybody's PR on mostly java stuff with running tests for 5 different python versions, so only one is tested as part of the PR check.
But after discussing with Jianfeng I think it makes sense to have a separate gradle build target that can be used during release so that all of them get tested before publishing to pipy, will do that.